spark/pom.xml

```diff
@@ -192,32 +191,47 @@
 
     <profiles>
 
+        <profile>
+            <id>spark-2.4</id>
+            <properties>
+                <spark.version>2.4.0</spark.version>
+                <protobuf.version>2.5.0</protobuf.version>
+                <py4j.version>0.10.7</py4j.version>
+            </properties>
+        </profile>
+
         <profile>
             <id>spark-2.3</id>
             <properties>
-                <spark.version>2.3.0</spark.version>
+                <spark.version>2.3.2</spark.version>
                 <protobuf.version>2.5.0</protobuf.version>
-                <spark.py4j.version>0.10.6</spark.py4j.version>
+                <py4j.version>0.10.6</py4j.version>
```

Comment:
Sorry, my fault, it should be 0.10.7 @HyukjinKwon

Reply:
Sure!